### PR TITLE
Schema SQL file for database.

### PR DESCRIPTION
**Pull request by indiah444**
- Created schema SQL file for database, creating the tables and relationships between them, columns and their data types, constraints etc.
- Could someone check over this file to make sure the constraints are all correct?

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    topic_id SMALLINT NOT NULL,
+    article_id BIGINT NOT NULL,
+    FOREIGN KEY (topic_id) REFERENCES topic(topic_id),
+    FOREIGN KEY (article_id) REFERENCES article(article_id)
+);
+
+CREATE TABLE subscriber (
+    subscriber_id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    subscriber_email VARCHAR(250) NOT NULL,
+    subscriber_first_name VARCHAR (100) NOT NULL,
```
Comment (megansam5) [inline database/schema.sql:42]: nice use of different lengths of varchar

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    subscriber_surname VARCHAR(100) NOT NULL
+);
+
+CREATE TABLE subscriber_topic_assignment (
+    subscriber_topic_assignment_id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    subscriber_id INT NOT NULL,
+    topic_id SMALLINT NOT NULL,
+    FOREIGN KEY (subscriber_id) REFERENCES subscriber(subscriber_id),
+    FOREIGN KEY (topic_id) REFERENCES topic(topic_id)
+);
```
Comment (megansam5) [inline database/schema.sql:52]: There is the possibility of including the basic seed data that we will need, e.g. all the topics, and maybe even the 2 sources we know for now.

Comment (indiah444) [reply database/schema.sql:52]: Oh, good point, I'll add that in now!

Comment (indiah444) [reply database/schema.sql:52]: Also, are there any other constraints that need to be added? Are there any columns that need a e.g. `UNIQUE` constraint?

Comment (indiah444) [reply database/schema.sql:52]: Okay, will do, thank you for your help!

**Review by megansam5 — APPROVED** [review]
Thanks for doing all the seeding!!

**Comment by indiah444** [comment]
This closes #1

**Pull request merged**